tests/customer-resource-show-test.js

```diff
@@ -237,4 +244,4 @@ describeApplication('CustomerResourceShow', () => {
       expect(ResourcePage.hasErrors).to.be.true;
     });
   });
-});
+}, describe);
```

Comment:
@wwilsman is this right/necessary? We want to use `describe` from `@bigtest/mocha` instead of `window.describe`?

Reply:
`describe` from `@bigtest/mocha` is forwarded straight from mocha. Which is actually `window.describe` in this case, so this shouldn't be necessary.